Libplanet.Explorer/Mutations/TransactionMutation.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using GraphQL;
+using GraphQL.Types;
+using Libplanet.Action;
+using Libplanet.Blockchain;
+using Libplanet.Explorer.GraphTypes;
+using Libplanet.Explorer.Interfaces;
+using Libplanet.Tx;
+
+namespace Libplanet.Explorer.Mutations
+{
+    public class TransactionMutation<T> : ObjectGraphType
+        where T : IAction, new()
+    {
+        private readonly IBlockChainContext<T> _context;
+
+        public TransactionMutation(IBlockChainContext<T> context)
+        {
+            _context = context ?? throw new ArgumentNullException(nameof(context));
+
+            Field<TransactionType<T>>(
```

Comment:
If I understood the logic well, it could return a non-null transaction value. Can you let me know what part you worried about?

Reply:
`ArgumentNullException` isn't related to `Field<TransactionType<T>>` but is relate to `context` argument. 😅